src/ResourceManager/Network/Commands.Network/help/Test-AzureRmNetworkWatcherConnectivity.md

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
@irrogozh you will also need to add a reference to this file in [`AzureRM.Network.md`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/help/AzureRM.Network.md)

(just follow the format of all other file references)